src/Resources/assets/package.json

```diff
@@ -15,6 +15,12 @@
                 "fetch": "eager",
                 "enabled": true
             },
+            "input-mask": {
+                "main": "controllers/input-mask_controller.js",
+                "webpackMode": "eager",
+                "fetch": "eager",
```

Comment:
If we can't utilize lazy loading here for some reason, it's even more important the used library is loaded asynchronously and not synchronously.